core/src/main/java/hudson/security/FullControlOnceLoggedInAuthorizationStrategy.java

```diff
@@ -92,7 +93,7 @@ public void setAllowAnonymousRead(boolean allowAnonymousRead) {
     @Restricted(NoExternalUse.class)
     public static Descriptor<AuthorizationStrategy> DESCRIPTOR;
 
-    @Extension
+    @Extension @Symbol("fullControlOnceLoggedIn")
```

Comment:
`NullSCM` seemed to be matching the Display rather than the class (`none` vs `null`) - yet this is doing the opposite (`fullControlOnceLoggedIn` vs `loggedInUsersCanDoAnything`?